pkg/admin/prerun/prerun.go

```diff
@@ -187,6 +189,11 @@ func (pr *PreRun) regularPrerun() error {
 
 		if migrationNeeded {
 			_ = migrationNeeded
+			stop, err := runMinimalMicroshift(pr.config)
+			if err != nil {
+				return fmt.Errorf("minimal MicroShift run failed to be ready: %w", err)
+			}
+			defer stop()
```

Comment:
We can put these lines and L220-L224 to a separate function, like a `migrateStorage` that would call `runMinimalMicroshift()` and run the migrator

Reply:
Sounds like a plan, do you want me to put that in a separate function in this PR, or do it in https://github.com/openshift/microshift/pull/1956 ?